lecture_02/assignment_01/farzaneh_eskandari/assignment-01

```diff
@@ -0,0 +1,25 @@
+"""Assignment 01: Project box to xy-plane
+"""
+
+from compas.geometry import Box
+from compas.geometry import Frame
+from compas.geometry import Projection
+from compas.artists import Artist
+from compas.datastructures import Mesh
+# Define a Frame, which is not in the origin and a bit tilted to the world frame
+frame = Frame([1, 3, 18], [18, 5, 12], [7, 8, 21])
+# Create a Box with that frame
+box = Box(frame, 3, 5, 7)
+# Create a Projection (can be orthogonal, parallel or perspective)
+point, normal = [0, 0, 0], [0, 0, 1]
+P = Projection.from_plane((point, normal))
+# Create a Mesh from the Box
+mesh = Mesh.from_shape(box)
+# Apply the Projection onto the mesh
+mesh_projected = mesh.transformed(P)
+# Create artists
+artist1 = Artist(box)
+artist2 = Artist(mesh_projected)
+# Draw
+artist1.draw()
+artist2.draw_edges(color="#00ff00")
```

Comment:
Sorry, the starting code was outdated, in the version of compas that we're using, you can pass colors as a list of RGB values, eg:
```suggestion
artist2.draw_edges(color=[0, 255, 0])
```